src/main/java/seedu/address/logic/commands/EditTaskCommand.java

```diff
@@ -42,7 +42,7 @@
             + PREFIX_TAG + "Homework";
 
     public static final String MESSAGE_EDIT_TASK_SUCCESS = "Updated Task: %1$s";
-    public static final String MESSAGE_NOT_EDITED = "At least one field to update must be provided.";
+    public static final String MESSAGE_NOT_EDITED = "A field has to be edited at least.";
```

Comment:
I think the statement, "At least one field to edit must be provided.", may sound better?🤔

Reply:
Just my opinion base on the EditCommand. 😆